definitions/procedures/restore/installer_reset.rb

```diff
@@ -0,0 +1,39 @@
+module Procedures::Restore
+  class InstallerReset < ForemanMaintain::Procedure
+    metadata do
+      description 'Run installer reset'
+
+      param :incremental_backup,
+            'Is the backup incremental?'
+    end
+
+    def run
+      with_spinner('Resetting') do |spinner|
+        if @incremental_backup
+          spinner.update('Skipping installer reset for incremental update')
+        else
+          spinner.update('Installer reset')
+          execute!(installer_cmd)
+        end
+      end
+    end
+
+    def installer_cmd
+      installer = "yes | #{feature(:installer).installer_command} "
+      installer << "--scenario #{feature(:installer).scenario_name} -v --reset"
```

Comment:
@johnpmitsch, why do we compute the scenario name here? At this point the configs are restored so the installer can figure it out on its own if `--scenario` is omitted. Or we can use `feature(:installer).last_scenario` if necessary.